e2e/device-client-app/src/autoscript.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ *
+ * Copyright © 2025 Ping Identity Corporation. All right reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ *
+ */
+
+import {
+  FRAuth,
+  FRStep,
+  CallbackType,
+  NameCallback,
+  TokenManager,
+  SessionManager,
+  Config,
+  PasswordCallback,
+} from '@forgerock/javascript-sdk';
+import { deviceClient } from '@forgerock/device-client';
+import { delay as rxDelay, mergeMap } from 'rxjs/operators';
+import { from } from 'rxjs';
```

Comment:
Since this is a new repo, I think it's worth asking if the use of RxJS is needed. We never really utilized it in the old repo in any way that provided enough value to offset its cost.

What do you all think? Should we just toss RxJS and move to async-await?

Reply:
From what I can tell it looks like rxjs was only used for the autoscripts in the old repo. So I guess the question is also do we plan to use rxjs for any other autoscript tests that we plan to port over?

I don't have a strong opinion but I do think rxjs improves readability by grouping together different steps in the test flow. However, that might not be a good enough reason to install an additional dependency.

Reply:
> [...] I do think rxjs improves readability by grouping together different steps in the test flow.

Okay, that's reasonable. Let's see what @ryanbas21 has to say. I personally don't mind RxJS, but I just want to make sure we critically consider it, rather than just copy it over.

Reply:
I'm fine with transitioning over to Effect. I was actually thinking about that, and wondered if you'd mention it. We are already using it, like you said, and will likely introduce more of it in our code. So, this would be a good place to start. 

@ryanbas21 Would you be able to give AJ a simple code translation of what she has to Effect?

@ancheetah I'll give you the last right of refusal. This is just an e2e test, so I don't want to overly stress how it's implemented. If you